engines/decidim_hospitalet-surveys/app/commands/decidim_hospitalet/surveys/admin/create_survey_result.rb

```diff
@@ -89,6 +89,13 @@ def existing_user
           ).first
         end
 
+        def default_user
+          Decidim::User.where(
```

Comment:
This should be a `find_or_create` instead. Otherwise, when deploying a new empty instance, even if the migration creates the data, this user wouldn't exist. This will probably never happen but IMO it's a best practice to do so.